examples/kubernetes/csi-attacher-glusterfsplugin.yaml

```diff
@@ -0,0 +1,80 @@
+# Copyright 2018 The Gluster CSI Authors.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This YAML file contains attacher & csi driver API objects that are necessary
+# to run external CSI attacher for glusterfs
+
+
+kind: Service
+apiVersion: v1
+metadata:
+  name: csi-attacher-glusterfsplugin
+  labels:
+    app: csi-attacher-glusterfsplugin
+spec:
+  selector:
+    app: csi-attacher-glusterfsplugin
+  ports:
+    - name: dummy
+      port: 12345
+
+---
+kind: StatefulSet
+apiVersion: apps/v1beta1
+metadata:
+  name: csi-attacher-glusterfsplugin
+spec:
+  serviceName: "csi-attacher"
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: csi-attacher-glusterfsplugin
+    spec:
+      serviceAccount: csi-attacher
+      containers:
+        - name: csi-attacher
+          image: quay.io/k8scsi/csi-attacher:v0.3.0
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
+              value: /var/lib/csi/sockets/pluginproxy/csi.sock
+          imagePullPolicy: "IfNotPresent"
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/lib/csi/sockets/pluginproxy/
+
+        - name: glusterfs
+          image: docker.io/madhupr001/glusterfs-csi-driver
```

Comment:
Please change this (and other occurrences) to `gluster/glusterfs-csi-driver`, which is going to be the correct location. Once the PR is merged we'll build and push this image to the dockerhub.

Though these are examples, we should be using the official images, not personal images. Using your own personal images is okay for testing.